packages/solid-query/src/createBaseQuery.ts

```diff
@@ -178,13 +190,17 @@ export function createBaseQuery<
     }
   })
 
-  onMount(() => {
-    observer.setOptions(defaultedOptions, { listeners: false })
-  })
```

Comment:
From what I can tell, this isn't doing anything. Removed to simplify and reduce the number of calls to `queryObserver.createResult`. LMK if there's a reason for this and I can revert np!